src/components/inputs/TextInput.js

```diff
@@ -209,6 +229,38 @@ export default class TextInput extends BaseInput {
     }
   }
 
+  renderTitle() {
+    const {floatingPlaceholder, title} = this.props;
+    if (!floatingPlaceholder && title) {
+      const capitalizedTitle = _.capitalize(title);
+      return (
+        <Text
+          style={this.styles.title}
+          text90
+        >
+          {capitalizedTitle}
+        </Text>
+      );
+    }
+  }
+
+  renderCharCounter() {
+    const {maxLength, characterCounter} = this.props;
+    if (maxLength && characterCounter) {
+      const {value} = this.state || '';
```

Comment:
This line is wrong, basically if the state is undefined you fallback to an empty string and then try to destruct "value" from it, theoretically it would throw and error if state was undefined.

Reply:
you mean this line: const {value} = this.state || ''; ? 
I already changed it to: const {value} = this.state;